### ISSUE TITLE
[Enhancement] Use same window session to download multiple courses

### ISSUE THREAD

**Issue by thefinalwolf** [ankitsejwal/Lyndor#32]
I think it would be a good idea to use the same window/session to download multiple courses instead of closing and re-signing in over and over again; it may speed up the download process.

**Comment by ankitsejwal** [comment]
Hi @thefinalwolf, to clarify: Do you mean once the terminal is launched the user is able to enter multiple URLs in the terminal and the program will run as if it is in BulkDownload mode?

**Comment by thefinalwolf** [comment]
Nah, I was thinking that once the exercise file is downloaded and there are more urls to process to keep the window open to process more downloads instead of closing and logging in multiple times; it might draw attention that an account is logging in 20 times every hour. I think it may be a low priority change though.

Btw, I'm experimenting with a headless download, but there are some kinks to work out. I'll be extremely busy indefinitely with my job starting next week though 😔

**Comment by ankitsejwal** [comment]
Got it, I'll look into it.
I know how it feels, it's difficult to keep up with the open-source stuff with the job, but anyways try to find some time if possible.
I tried to go headless earlier as well, but it seems Lynda got their real exercise file URLs hidden which only seems to populate with Ajax once we click on the exercise file link.
If you come around a solution or find something approachable do let me know.
